apps/events/EventsDatabase.ts

```diff
@@ -0,0 +1,228 @@
+import { createClient } from '@supabase/supabase-js';
+import { AddEventsInterface, EventsInterface } from './EventsInterface';
+
+const supabaseUrl = 'https://igfaeuwzpmjmcbfwstzu.supabase.co';
+const supabaseKey = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImlnZmFldXd6cG1qbWNiZndzdHp1Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2OTgyNzIzOTUsImV4cCI6MjAxMzg0ODM5NX0.mnmXiwnQSlT7XXcT6V4YhsaWmQsj23VwBgcFF9EBxtc';
+const supabase = createClient(supabaseUrl, supabaseKey);
+
+// Adds an event to the database
+export async function addEvent(eventData: AddEventsInterface): Promise<void> {
+    try {
+        const { data, error } = await supabase
+            .from('Events')
+            .insert([eventData]);
+
+        if (error) {
+            console.error('Error inserting event:', error);
+        } else {
+            console.log('Event inserted successfully:', data);
+        }
+    } catch (error: any) {
+        console.error('Error:', error);
+    }
+}
+
+export async function deleteEvent(event: EventsInterface): Promise<void> {
```

Comment:
Instead of EventsInterface, these should have type Event

Reply:
Done